source/adios2/helper/adiosCUDA.h

```diff
@@ -10,27 +10,19 @@ namespace adios2
 {
 namespace helper
 {
-
-#ifdef _WIN32
-#define ADIOS2_EXPORT __declspec(dllexport)
-#else
-#define ADIOS2_EXPORT __attribute__((visibility("default")))
-#endif
-
```

Comment:
The errors are different than what I am seeing on Summit because I think you used a older version of my commits. The `ADIOS2_EXPORT` should not be deleted from this file and it should appear in the `adiosKokkos.h` header file as well